niceml/data/normalization/dataframe.py

```diff
@@ -29,10 +34,73 @@ def normalize_col(
 
     divisor = max_val - min_val
 
+    if divisor == 0:
+        if min_val == max_val:
+            divisor = 1
+        else:
+            raise ZeroDivisionError
```

Comment:
Why do we have to raise a ZeroDivisionError here?
The divisor can only be 0, if min_val == max_val. So the ZeroDivisionError should never be raised.